state/open.go

```diff
@@ -32,13 +30,10 @@ type OpenParams struct {
 	// ControllerModelTag is the tag of the controller model.
 	ControllerModelTag names.ModelTag
 
-	// MongoInfo is the mongo.MongoInfo used for dialling the
-	// Mongo connection.
-	MongoInfo *mongo.MongoInfo
-
-	// MongoDialOpts is the mongo.DialOpts used to control how
-	// Mongo connections are made.
-	MongoDialOpts mongo.DialOpts
+	// MongoSession is the mgo.Session to use for storing and
+	// accessing state data. The caller remains responsible
+	// for closing the session; Open will copy it.
```

Comment:
s/the/the passed in session?

Not sure if that clarification is needed? Makes sense to make the change to me anyway. YMMV

Reply:
s/the/this/ like in the other case